src/Ship.java

```diff
@@ -12,7 +12,7 @@ public Ship(int life) {
 	// in the observer pattern
 	//
 	public void update() throws Exception {
-
+	  hit();
```

Comment:
A little inconsistent with the codebase. Notice how every method call and test utilizes the `this` keyword? Though  not required in Java in some contexts, it's best to keep the same conventions throughout the entire project.